puan/modules/configurator/__init__.py

```diff
@@ -198,7 +198,7 @@ def from_json(data: dict, class_map: list) -> "Xor":
                 variable=data.get("id", None)
             )
         else:
-            return pg.Xor.from_json(data, class_map)
+            return Xor(*pg.Xor.from_json(data, class_map).propositions, variable=pg.Xor.from_json(data, class_map).variable)
```

Comment:
```suggestion
            return Xor(
                *map(functools.partial(pg.from_json, class_map=class_map), data.get('propositions', [])),
                default=[],
                variable=data.get("id", None)
            )
```
To skip parsing twice